src/content/docs/apm/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected.mdx

```diff
@@ -92,6 +92,19 @@ To view expected errors, [query your data](/docs/using-new-relic/data/understand
 * To view charts of expected errors, create a query for the [`error.expected`](/docs/insights/insights-data-sources/default-events-attributes/apm-default-event-attributes#transerror-expected) attribute.
 * To create [alert conditions for NRQL queries](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries), use the `error.expected` attribute.
 
+## Error fingerprinting with errors inbox [#error-fingerprinting]
+
+Errors inbox intelligently groups error occurrences to reduce noise and highlight important errors. 
+
+Error events get grouped into an error group when they share the [same fingerprint](/docs/errors-inbox/errors-inbox/#how-groups-work). While our managed rules are able to provide automatic error grouping based on a predefined set of patterns, it is impossible to recognize every possible combination. For this reason, customers can also set their own fingerprint via a callback function if they find that our managed rules are not grouping occurrences accurately. 
+
+To configure a fingerprint, see the agent-specific documentation: 
```

Comment:
Perhaps "To configure a fingerprint callback," or "To configure custom fingerprint logic," as there is one singular callback but many fingerprints involved.

Reply:
Thanks @fallwith !